docs/simulators/Reef3D.md

```diff
@@ -133,28 +134,33 @@ T 10 0
 W 22 -9.81
 ```
 
-Observe that parameter ```M 10```, which controls the degree of parallism, is
+Observe that parameter ```M 10```, which controls the level of parallism, is
 set to 4 threads/vCPUs, a very low number. Depending on the number of vCPUs we
 effectively wish to use for running the simulation, we will need to manually
 change  ```M 10``` on **both files** to match the specs of corresponding the VM. 
-Because this is already a reasoably heavy simulation, we will be using GCP VMs
-of the c3d family, supported by last-generation AMD chips. More specifically,
-we will be using ```c3d-standard-90``` machines with 90 vCPU. So, we will be
-setting ```M 10 90``` on both files.
-
-Also, Reef3D produces a huge amount of data. As it is currently configured, this
-simulation would produce several dozen gigabytes of data. To reduce that amount
-of data produced, we can reduce the maximum modeled time (```N 41```) and
-the rate at which Paraview data is being produced (```P 10```).
-
-So, for a reducing the amount of data generated, we will be reducing the rate
-at which Paraview information is going to be generated to "onyy" 25 frames per
-second, that is, we will set ```P 10 0.04```. 
-
-We are not going to reduce the maximum modeled time but, instead, we will add
-extra disk capacity to ensure we have enough space to write all the data will be
-generated. Therefore, we will request our machine to be equipped
-with **20GB** just for data, using the ```data_disk_gb``` parameter of the 
+We will be using GCP VMs of the c2d family, which tend to provide a very good 
+price-performance point, especially when you run them in spot mode. More 
+specifically, we will be using the larger ```c2d-highcpu-112``` machines with
+112 vCPUs and 2GB of RAM per vCPU (this simulation will not require more than
+224GB of RAM). 
+
+We will be setting ```M 10 56``` on both files. Note that this is half the
+available number of vCPUs available on the VM (112!). Actually, running on only
+half of the vCPUs may provide significantly faster simulation times than running
+on all the vCPUs *for some simulators / VM types*. This has to do with the fact
+that these VMs run on processors with hyperthreading, running two threads per 
+physical core. One of the issues with hyperthreading is that it increases
+thread competition for cache (which does is fixed), and this may lead to some
+contention issues for I/O heavy simulators, as seems to be the case for Reef3D.
```

Comment:
```suggestion
that these VMs run on processors with hyperthreading, meaning that two threads will run per 
physical core. One of the issues with hyperthreading is that it increases
thread competition for cache (which is fixed), and this may lead to some
contention issues for I/O heavy simulators, as seems to be the case for Reef3D.
```

Reply:
Done